.github/workflows/conjecture.yml

```diff
@@ -0,0 +1,32 @@
+name: Conjecture-Rust CI
+
+on:
+  push:
+    branches: [ master ]
+    paths:
```

Comment:
It should also be in sync with https://github.com/HypothesisWorks/hypothesis/blob/master/.github/workflows/main.yml#L3-L8

Because right now we have Python jobs running for no reason 😞

Reply:
Done! Added inverted versions to `main.yml`